### PR TITLE
add rotateOnWorldAxis method to Object3D

### LINKED ISSUE #12336: [suggestion] add a rotateOnWorldAxis helper
I keep seeing global-rotation related problems and questions pop up, which makes sense, as three's "simple" rotation methods are local and grokking the quaternion math involved in conversions is nontrivial, especially for beginners.

Would the maintainers ( @mrdoob  / @WestLangley / whoever's making feature decisions) be open to a helper method that simplifies rotating in world axis? To conform with the existing (local) rotateOnAxis, The API could be something like:

`Object3D.rotateOnWorldAxis([Vector3], [radians])
`

and the implementation similar to the solution I provided [here](https://stackoverflow.com/questions/46571648/aframe-rotate-down).

I think this would really benefit the community - I know I, for one, scratch my head every time I have to reimplement world rotation...

### PR DESCRIPTION
**Pull request by michaelybecker**
Added a builtin world space rotation method per the recent discussion/issue: closes #12336.

### CODE REVIEW

File: src/core/Object3D.js [hunk 1]
```diff
@@ -170,6 +170,26 @@ Object.assign( Object3D.prototype, EventDispatcher.prototype, {
...
 
+	rotateOnWorldAxis: function () {
+		
+		// rotate object on axis in world space
+		// axis is assumed to be normalized
+		// method assumes no rotated parent
+
+		var q1 = new Quaternion();
+		
+		return function rotateOnWorldAxis(axis, angle) {
```
Comment (WestLangley) [inline src/core/Object3D.js]: please follow three.js white-space formatting

Comment (MichaelHazani) [reply src/core/Object3D.js]: Sorry, I did my best but missed the argument spacing specification. Fixed now.

Comment (WestLangley) [reply src/core/Object3D.js]: @Mugen87 may be able to answer that. In the mean time, see http://zz85.github.io/mrdoobapproves/.

**Comment by MichaelHazani** [comment]
Thanks, I was feeling brave due to it being a small addition but I'll make
sure to do so from now on.

File: src/core/Object3D.js (hunk 1 above)
Comment (MichaelHazani) [reply src/core/Object3D.js]: Thanks, I was feeling brave due to it being a small change but I'll make sure to do so moving forward.

**Comment by WestLangley** [comment]
Thanks!

**Review by Mugen87 — APPROVED** [review]

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**